### ISSUE TITLE
A job not yet reviewed by QA is removed from my agenda

### ISSUE THREAD

**Issue by victornoel** [zerocracy/farm#2070]
The issue https://github.com/llorllale/cactoos-matchers/issues/119 was assigned to me and present in my agenda, when it was closed, a QA was assigned, but at the current time the QA review hasn't been done.

But the job was already removed from my agenda. Also if you look at 0crat comment (https://github.com/llorllale/cactoos-matchers/issues/119#issuecomment-499735061), it says:
> Job #119 is not in the agenda of @victornoel/z, can't inspect

I believe there is a problem here: the job should be removed from the agenda only after review by QA has been done.

For the record, I have another job (https://github.com/yegor256/cactoos-http/issues/42) that was closed and is not yet reviewed by QA, and it is still in my agenda as expected:
![image](https://user-images.githubusercontent.com/160975/59092140-22a1f700-8911-11e9-9968-b44e51b9f21d.png)

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by victornoel** [comment]
@g4s8 this happened again with https://github.com/g4s8/ghman/pull/26

**Comment by victornoel** [comment]
@g4s8 can this at least be put in scope? I think this should be solved, I loose so much time keeping track of the issue I closed but that QA hasn't evaluated yet because 0crat is not doing it for me.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#2070](https://github.com/zerocracy/farm/issues/2070), see [§21](http://www.zerocracy.com/policy.html#21); @g4s8[/z](https://www.0crat.com/u/g4s8) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by victornoel** [comment]
@g4s8 for the record we experience also this deteministically when there are no QA available in a project : https://github.com/yegor256/cactoos/issues/1152

The job is removed from scope, from agenda, even though ARC has not evaluated quality. Maybe this is a bit different, but it concerns the same pain: the fact that as a zerocracy user, I have to do the job that I am expecting of zerocracy, i.e. managing my jobs.

**Comment by g4s8** [comment]
@rultor release, tag=`0.49.9`

**Comment by g4s8** [comment]
@rultor release, tag=`0.49.9`

**Comment by g4s8** [comment]
@victornoel this is another bug with yegor256/cactoos#1152 ticket, this fix https://github.com/zerocracy/farm/commit/96d147b638eda205cc43de66258d915551efb336 won't solve it

**Comment by g4s8** [comment]
@victornoel should be fixed now

**Comment by victornoel** [comment]
> @victornoel this is another bug with yegor256/cactoos#1152 ticket, this fix 96d147b won't solve it

@g4s8 do you want me to open another ticket or do you want to cover this here? Both problems are related to the same pain/requirement even though they each have their fix.

**Comment by g4s8** [comment]
@victornoel the bug with  yegor256/cactoos#1152 is another because 0crat just closed the job without payment and not put it into QA review.

**Comment by victornoel** [comment]
@g4s8 thank you, I have opened #2155 for this.

**Comment by 0crat** [comment]
Job [#2070](https://github.com/zerocracy/farm/issues/2070) is not in the agenda of @g4s8[/z](https://www.0crat.com/u/g4s8), can't retrieve data and time of add

**Comment by 0crat** [comment]
Job was finished in 27 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#2070](https://github.com/zerocracy/farm/issues/2070) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)
